src/components/map/map-tooltip.js

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
Is this used anywhere?

Reply:
I forgot to add this to MapContainer. this will replace _renderMapPopover method

Reply:
I will remove it from this diff